src/main.c

```diff
@@ -129,31 +129,30 @@ void create_thread(OSThread *thread, OSId id, void (*entry)(void *), void *arg,
 
 void main_func(void) {
     osInitialize();
-    create_thread(&D_801524C0, 1, thread1_idle, NULL, &D_80154670, 100);
-    osStartThread(&D_801524C0);
+	//! Why is this using the gVideoThread as idle stack?
+    create_thread(&gIdleThread, 1, thread1_idle, NULL, &gVideoThread, 100);
```

Comment:
Is it possible they chose gVideoThread due to the below size requirements? Looks like the stack size of the idlethread is set to the size of the gVideoThread. I tried checking the original asm but it likely no longer exists. Could double check that `&gVideoThread` set as the stack wasn't a mistake on our part or the wrong name.
```
Similarly, the stack is initialized to the location pointed to by the sp argument. Note that stacks grow backwards (towards smaller addresses) with the MIPS compiler tools. The memory region assigned to the stack must be of least size OS_MIN_STACKSIZE in bytes. The maximum size of the stack must be the sum of the stack frame sizes of its maximum procedure call depth plus 16. Stacks must also be 64-bit aligned, so they should be declared as 'long long int' and type-casted when passed to this function. Use the stacktool program to help compute stack sizes.
```